.github/workflows/ISSUE_TEMPLATE/config.yml

```
@@ -0,0 +1 @@
+blank_issues_enabled: false
```

actionlint (1.7.7)

unexpected key "blank_issues_enabled" for "workflow" section. expected one of "concurrency", "defaults", "env", "jobs", "name", "on", "permissions", "run-name"

(syntax-check)